packages/x-charts-pro/src/ChartContainerPro/ChartContainerPro.tsx

```diff
@@ -35,15 +38,15 @@ const ChartContainerPro = React.forwardRef(function ChartContainer(
     chartsSurfaceProps,
     xAxis,
     yAxis,
-  } = useChartContainerProps(props, ref);
+  } = useChartContainerProps(baseProps, ref);
```

Comment:
Same comment as for the `useResponsiveChartContainerProProps`.

The `useChartContainerProProps` could include the `useChartContainerProps` to simplify the usage